### PR TITLE
GH-1096 Standardizes Column & Bar

### PR DESCRIPTION
**Pull request by mlzummo**
@GordonSmith @dtsnell4 @jbrundage - Please Review

Standardizes Column & Bar
Fixes GH-1096

Signed-off-by: Mathew Zummo <mzummo@gmail.com>

### CODE REVIEW

**Comment by mlzummo** [comment]
@GordonSmith - Can you please review and merge?

File: src/amchart/CommonSerial.js [hunk 1]
```diff
@@ -201,7 +201,7 @@
 
         this._chart.type = "serial";
         this._chart.startDuration = this.startDuration();
-        this._chart.rotate = this.orientation() === "vertical"; // this messes up the hover over things
+        this._chart.rotate = this.orientation() === "horizontal"; // this messes up the hover over things
```
Comment (dtsnell4) [inline src/amchart/CommonSerial.js]: How does this mess up the hover?  it looked normal to me...

**Comment by dtsnell4** [comment]
Looks good to me

File: src/amchart/CommonSerial.js (hunk 1 above)
Comment (mlzummo) [inline src/amchart/CommonSerial.js]: thats an old comment i need to remove

**Comment by mlzummo** [comment]
@GordonSmith Please Review and pull

**Comment by mlzummo** [comment]
@GordonSmith ... please take a look at the below comment/issue
https://github.com/hpcc-systems/Visualization/issues/1182

**Comment by mlzummo** [comment]
@GordonSmith please review and pull think were all set here

**Pull request merged**